ImageGallery/src/org/sleuthkit/autopsy/imagegallery/utils/TaskUtils.java

```diff
@@ -42,8 +47,26 @@ protected T call() throws Exception {
         };
     }
 
-   public  static ListeningExecutorService getExecutorForClass(Class<?> clazz) {
+    public static ListeningExecutorService getExecutorForClass(Class<?> clazz) {
         return MoreExecutors.listeningDecorator(Executors.newSingleThreadExecutor(
                 new ThreadFactoryBuilder().setNameFormat("Image Gallery " + clazz.getSimpleName() + " BG Thread").build()));
     }
+
```

Comment:
This is just a utils class but I think adding method headers here would be useful